tests/integration/test_examples.py

```diff
@@ -114,3 +115,16 @@ def test_csv(csv_cleanup):
     """
     # Run entire scan.py script
     runpy.run_path("csv_output.py")
+
+    # Check csv file is created
+    assert os.path.exists("csv_output_large_tokamak_MFILE.csv")
+
+    # Read in the csv file created by test and check it contains values
+    readcsv = pandas.read_csv("csv_output_large_tokamak_MFILE.csv")
+    values = readcsv["Value"]
+    check = False
+    for item in values:
+        check = isinstance(item, float)
+        if check is True:
+            break
+    assert check
```

Comment:
This is much better. But this will pass if any one of the values is a float. Can you check that all the values in the column are floats, without using a for loop?

Reply:
Hi Clair. What I was trying to get at (and should have explained better) was if this check could be vectorised (i.e. the entire array operated on, rather than looped over one element at a time).